2020/27xxx/CVE-2020-26074.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
Expected location (2020/26xxx/CVE-2020-26074.json) of file does not match actual location (2020/27xxx/CVE-2020-26074.json)

Reply:
I had a typo in a script and put it in the 26xxx directory. I will create a new pull request to address this.